src/commands/deploy.rs

```diff
@@ -295,6 +295,7 @@ impl DeployCommand {
                 &app_base_url,
                 &bindle_id.version_string(),
                 self.readiness_timeout_secs,
+                false,
```

Comment:
Bit of a nit I guess, but these constants are hard to understand at the call site.  If it were me, and if I were feeling both energetic and smugly righteous, I would make an enum a la `Destination::{Cloud|Platform}`.  But I completely sympathise if you go 'eh not worth it.'

Reply:
How convenient. You've caught me in an energetic and smugly righteous mood. 😄

Reply:
Random booleans are the death of a beautiful codebase.

Reply:
PREACH IT

Reply:
```rust
rand::random<bool>()
```